### PR TITLE
config size adaptive icon

### PR DESCRIPTION
**Pull request by rs-tupa1**

### CODE REVIEW

File: lib/android.dart [hunk 1]
```diff
@@ -106,6 +106,7 @@ void createAdaptiveIcons(
   }
 
   // Create adaptive icon foreground images
+  printStatus('tupa1 changed');
```
Comment (MarkOSullivan94) [inline lib/android.dart:109]: Please remove

File: lib/android.dart [hunk 2]
```diff
@@ -188,6 +189,7 @@ void _createAdaptiveBackgrounds(
 
   // creates a png image (ic_adaptive_background.png) for the adaptive icon background in each of the locations
   // it is required
+  printStatus('tupa1 changed');
```
Comment (MarkOSullivan94) [inline lib/android.dart:192]: Please remove

File: lib/android.dart [hunk 3]
```diff
@@ -17,11 +17,11 @@ class AndroidIconTemplate {
...
-  AndroidIconTemplate(directoryName: 'drawable-mdpi', size: 108),
-  AndroidIconTemplate(directoryName: 'drawable-hdpi', size: 162),
-  AndroidIconTemplate(directoryName: 'drawable-xhdpi', size: 216),
-  AndroidIconTemplate(directoryName: 'drawable-xxhdpi', size: 324),
-  AndroidIconTemplate(directoryName: 'drawable-xxxhdpi', size: 432),
+  AndroidIconTemplate(directoryName: 'drawable-mdpi', size: 24),
+  AndroidIconTemplate(directoryName: 'drawable-hdpi', size: 36),
+  AndroidIconTemplate(directoryName: 'drawable-xhdpi', size: 48),
+  AndroidIconTemplate(directoryName: 'drawable-xxhdpi', size: 72),
+  AndroidIconTemplate(directoryName: 'drawable-xxxhdpi', size: 96),
```
Comment (MarkOSullivan94) [inline lib/android.dart:24]: Can you link to source why this needs to change to this size?

**Comment by RatakondalaArun** [comment]
Hi @rs-tupa1 Can you please resolve the above mentioned comments

**Comment by MarkOSullivan94** [comment]
Cannot approve this without documentation which states these changes are required

**Pull request closed without merging**